package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "graphql-query-whitelist",
-  "version": "0.3.1",
+  "version": "0.3.2",
```

Comment:
Since you're adding a new feature, you should bump minor:

>Given a version number MAJOR.MINOR.PATCH, increment the:
>
>MAJOR version when you make incompatible API changes,
>**MINOR version when you add functionality in a backwards-compatible manner**, and
>PATCH version when you make backwards-compatible bug fixes.

Source: http://semver.org/